charts/astronomer/templates/houston/cronjobs/houston-expire-deployments-cronjob.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         metadata:
           labels:
             tier: astronomer
-            component: houston
+            component: houston-expire-deployments
```

Comment:
writing the unit test was helpful, it made me notice that we also need this label on the pod, just just the cronjob